foodsaving/invitations/api.py

```diff
@@ -20,6 +20,13 @@ def has_object_permission(self, request, view, obj):
         return not obj.group.is_member(request.user)
 
 
+class CanResend(BasePermission):
+    from datetime import timedelta
```

Comment:
Can you move this import to the top of the file? We usually don't have local imports.

(And we usually use `relativedelta` so you could do the same for consistency, but `timedelta` does the same in this case.)

Reply:
Yes, sure! I just did the import this way because the module was only used by this specific class, so I thought it was best this way. I'll make the change, no problem.